README.md

```diff
@@ -1,52 +1,242 @@
+type TWEventType =
+  | 'CALENDAREVENT.CREATED'
+  | 'CALENDAREVENT.DELETED'
+  | 'CALENDAREVENT.REMINDER'
+  | 'CALENDAREVENT.UPDATED'
+  | 'CARD.CREATED'
+  | 'CARD.DELETED'
+  | 'CARD.UPDATED'
+  | 'COLUMN.CREATED'
+  | 'COLUMN.DELETED'
+  | 'COLUMN.UPDATED'
+  | 'COMMENT.CREATED'
+  | 'COMMENT.DELETED'
+  | 'COMMENT.UPDATED'
+  | 'COMPANY.CREATED'
+  | 'COMPANY.DELETED'
+  | 'COMPANY.UPDATED'
+  | 'EXPENSE.CREATED'
+  | 'EXPENSE.DELETED'
+  | 'EXPENSE.UPDATED'
+  | 'FILE.CREATED'
+  | 'FILE.DELETED'
+  | 'FILE.DOWNLOADED'
+  | 'FILE.TAGGED'
+  | 'FILE.UNTAGGED'
+  | 'FILE.UPDATED'
+  | 'INVOICE.COMPLETED'
+  | 'INVOICE.CREATED'
+  | 'INVOICE.DELETED'
+  | 'INVOICE.REOPENED'
+  | 'INVOICE.UPDATED'
+  | 'LINK.CREATED'
+  | 'LINK.DELETED'
+  | 'LINK.TAGGED'
+  | 'LINK.UNTAGGED'
+  | 'LINK.UPDATED'
+  | 'MESSAGE.CREATED'
+  | 'MESSAGE.DELETED'
+  | 'MESSAGE.TAGGED'
+  | 'MESSAGE.UNTAGGED'
+  | 'MESSAGE.UPDATED'
+  | 'MESSAGEREPLY.CREATED'
+  | 'MESSAGEREPLY.DELETED'
+  | 'MESSAGEREPLY.UPDATED'
+  | 'MILESTONE.COMPLETED'
+  | 'MILESTONE.CREATED'
+  | 'MILESTONE.DELETED'
+  | 'MILESTONE.REMINDER'
+  | 'MILESTONE.REOPENED'
+  | 'MILESTONE.TAGGED'
+  | 'MILESTONE.UNTAGGED'
+  | 'MILESTONE.UPDATED'
+  | 'NOTEBOOK.CREATED'
+  | 'NOTEBOOK.DELETED'
+  | 'NOTEBOOK.TAGGED'
+  | 'NOTEBOOK.UNTAGGED'
+  | 'NOTEBOOK.UPDATED'
+  | 'PORTFOLIOBOARD.CREATED'
+  | 'PORTFOLIOBOARD.DELETED'
+  | 'PORTFOLIOBOARD.UPDATED'
+  | 'PORTFOLIOCARD.CREATED'
+  | 'PORTFOLIOCARD.DELETED'
+  | 'PORTFOLIOCARD.MOVED'
+  | 'PORTFOLIOCARD.REOPENED'
+  | 'PORTFOLIOCARD.UPDATED'
+  | 'PORTFOLIOCOLUMN.CREATED'
+  | 'PORTFOLIOCOLUMN.DELETED'
+  | 'PORTFOLIOCOLUMN.UPDATED'
+  | 'PROJECT.ARCHIVED'
+  | 'PROJECT.COMPLETED'
+  | 'PROJECT.COPIED'
+  | 'PROJECT.CREATED'
+  | 'PROJECT.DELETED'
+  | 'PROJECT.REOPENED'
+  | 'PROJECT.TAGGED'
+  | 'PROJECT.UNTAGGED'
+  | 'PROJECT.UPDATED'
+  | 'PROJECTUPDATE.CREATED'
+  | 'PROJECTUPDATE.DELETED'
+  | 'PROJECTUPDATE.UPDATED'
+  | 'RISK.CREATED'
+  | 'RISK.DELETED'
+  | 'RISK.UPDATED'
+  | 'ROLE.CREATED'
+  | 'ROLE.DELETED'
+  | 'ROLE.UPDATED'
+  | 'STATUS.CREATED'
+  | 'STATUS.DELETED'
+  | 'STATUS.UPDATED'
+  | 'TASK.COMPLETED'
+  | 'TASK.CREATED'
+  | 'TASK.DELETED'
+  | 'TASK.MOVED'
+  | 'TASK.REMINDER'
+  | 'TASK.REOPENED'
+  | 'TASK.TAGGED'
+  | 'TASK.UNTAGGED'
+  | 'TASK.UPDATED'
+  | 'TASKLIST.COMPLETED'
+  | 'TASKLIST.CREATED'
+  | 'TASKLIST.CREATEDFROMTEMPLATE'
+  | 'TASKLIST.DELETED'
+  | 'TASKLIST.REOPENED'
+  | 'TASKLIST.UPDATED'
+  | 'TIME.CREATED'
+  | 'TIME.DELETED'
+  | 'TIME.TAGGED'
+  | 'TIME.UNTAGGED'
+  | 'TIME.UPDATED'
+  | 'USER.CREATED'
+  | 'USER.DELETED'
+  | 'USER.UPDATED'
+```
+
+Events require that an integration webhook is configured in Teamwork. 
+
+The connector triggers events of many types, you can get more details about them [here](https://developer.teamwork.com/projects/api-v1/ref)
+
+
+_Example:_
+
+```typescript
+connector.on({ type: 'TASK.UPDATED' }, async (event, app) => {
+  console.log('TASK.UPDATED')
```

Comment:
This is the same example from the top of the readme - so if we change the log message there.- please change here too